src/compas_ags/utilities/equilibrium.py

```diff
@@ -43,6 +50,80 @@ def check_deviations(form, force, tol=10e-3):
         else:
             checked = False
```

Comment:
break?

Reply:
Ok I will add the break, this should therefore not be useed to update the edge_attribute for the angle deviation 'a', we can create another function later for that.